lib/evss/service.rb

```diff
@@ -5,10 +5,7 @@
 module EVSS
   class Service < Common::Client::Base
     def headers_for_user(user)
-      @headers_by_user ||= Hash.new do |h, key|
-        h[key] = EVSS::AuthHeaders.new(user).to_h
-      end
-      @headers_by_user[user.uuid]
+      EVSS::AuthHeaders.new(user).to_h
```

Comment:
As I understand, passing a block to `Hash.new` ensures default values for the hash should someone attempt to read a non-existent key.  Since this is no longer memoized, is it correct to assume that we don't need this functionality anymore?

Reply:
I'd recommend including the author of this logic on this review.

Reply:
The author was me 😀 . This was part of getting EVSS on the common client, currently on letters has been migrated over. The original EVSS service doesn't memoize the headers so it's basically rolling it back to it's original state.